Makefile

```diff
@@ -133,7 +134,10 @@ install-go-test-tools:
 
 go-tests:
```

Comment:
This now covers EVERY possible go test file in the repo. Before there were some issues due to multiple go.mods being present and test runs being hidden on CI.

Now this enable for a sane framework